testsuite/E22.py

```diff
@@ -76,6 +76,10 @@
 i=i+ 1
 #: E225 E225
 i=i +1
+#: E225
+i = 1and 1
+#: E225
+i = 1or 0
```

Comment:
here's a few more:

```python
1is 1
1not in []
1in []
```

Reply:
Added support for 'is' and 'not', updating for the two-word 'not in' is beyond me right now (07:30 after a long haul flight...)

Reply:
missed `not` ;)